views/login_error.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Username or password is incorrect</h1>
```

Comment:
to make the code even more in depth you could try to do it where it states exactly whats wrong rather then username or password